pkg/config/services.go

```diff
@@ -101,6 +106,52 @@ func (s *IndexingServiceConfig) ToAppConfig() (app.IndexingServiceConfig, error)
 	return out, nil
 }
 
+type EgressTrackingServiceConfig struct {
+	DID   string `mapstructure:"did" validate:"required" flag:"egress-tracking-service-did" toml:"did,omitempty"`
+	URL   string `mapstructure:"url" validate:"required,url" flag:"egress-tracking-service-url" toml:"url,omitempty"`
+	Proof string `mapstructure:"proof" flag:"egress-tracking-service-proof" toml:"proof,omitempty"`
+}
+
+func (c *EgressTrackingServiceConfig) Validate() error {
+	return validateConfig(c)
+}
+
+func (c *EgressTrackingServiceConfig) ToAppConfig() (app.EgressTrackingServiceConfig, error) {
+	sdid, err := did.Parse(c.DID)
+	if err != nil {
+		return app.EgressTrackingServiceConfig{}, fmt.Errorf("parsing egress tracking service DID: %w", err)
+	}
+	surl, err := url.Parse(c.URL)
+	if err != nil {
+		return app.EgressTrackingServiceConfig{}, fmt.Errorf("parsing egress tracking service URL: %w", err)
+	}
+	schannel := ucanhttp.NewHTTPChannel(surl)
```

Comment:
Method changed to `ucanhttp.NewChannel`

Reply:
yeah, my AI mate invented that one. I'll fix it along with the call to `RequestProofs` in [line 312](https://github.com/storacha/piri/pull/243/files#diff-efa377e335526be56a9058067dbcb4083c5764e9f7b4977d7035fb02248fc2c4R312) that doesn't exist yet.